internal/manifests/gateway/openshift.go

```diff
@@ -222,6 +222,7 @@ func NewOpaContainer(ctrlConfig configv1alpha1.ProjectConfig, tenants v1alpha1.T
 	var args = []string{
 		"--log.level=warn",
 		"--opa.admin-groups=system:cluster-admins,cluster-admin,dedicated-admin",
+		"--opa.matcher=kubernetes_namespace_name",
```

Comment:
Does this setting change anything else, except for adding the list of namespaces to the Go context in the gateway?

Reply:
I am not sure, but I don't think it will be, it should be just an optional parameter